src/components/clusterWizard/ClusterWizardStep.tsx

```diff
@@ -16,36 +26,70 @@ const wizardSteps: ClusterWizardStepsType[] = [
   'review',
 ];
 
-const ClusterWizardStep: React.FC<ClusterWizardStepProps> = ({ footer, children }) => {
+const NavItem: React.FC<WizardNavItemProps & { isValid?: () => boolean }> = ({
+  content,
+  isDisabled,
+  isCurrent,
+  isValid = () => true,
```

Comment:
Any reason why `isValid` is not a boolean?

Reply:
Just performance reason, to postpone validation function call.